smart_campus/app/views.py

```diff
@@ -334,6 +365,7 @@ def get_all_rewards(request):
 @csrf_exempt
 def get_all_stations(request):
     """API for retrieving contents of all Stations"""
+    print(request.get_host())
```

Comment:
Why is this `print` statement needed?

Reply:
Just for debug, forget to delete it.
Fixed it.